qemu/tests/driver_in_use.py

```diff
@@ -84,12 +84,9 @@ def run_subtest(sub_type):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     error_context.context("Boot guest with %s device" % driver, logging.info)
-    session = vm.wait_for_login(timeout=timeout)
 
-    error_context.context("Enable %s driver verifier in guest" % driver,
-                          logging.info)
-    session = utils_test.qemu.setup_win_driver_verifier(session,
-                                                        driver, vm, timeout)
+    utils_test.qemu.setup_win_driver_verifier(session, driver, vm)
```

Comment:
Need to remove parameter session here, as it has been dropped in avocado-vt#600.